Add ossec.conf references (#6249)
wazuh/wazuh-documentation: source/cloud-security/gcp/supported-services/access_logs.rst

Google Cloud Storage offers `usage logs and storage logs <https://cloud.google.com/storage/docs/access-logs>`__, also known as access logs, in the form of CSV files that can be downloaded. Usage logs provide information for all of the requests made on a specified bucket and are created hourly. Storage logs provide information about the storage consumption of that bucket for the last day and are created daily. Once set up, usage logs and storage logs are automatically created as new objects in the specified bucket.

> To process Storage and Access logs, Wazuh makes use of the **gcp-bucket** module. To configure manager and agent, check the :doc:`/user-manual/reference/ossec-conf/index` documentation. You can find information regarding the configuration of this module in the :ref:`gcp-bucket configuration reference <gcp-bucket>`.

Reviewer: ```suggestion
To process Storage and Access logs, Wazuh makes use of the ``gcp-bucket`` module. Configure the ``gcp-bucket`` module either in the Wazuh manager or the Wazuh agent by modifying the :doc:`ossec.conf </user-manual/reference/ossec-conf/index>` configuration file. Check the :ref:`gcp-bucket configuration reference <gcp-bucket>` to learn more.
```
